rules/windows/credential_access_web_config_file_access.toml

```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2025/07/23"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2025/07/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects unusual access to the web.config file, which contains sensitive credential information such as database
+connection strings, machineKey validation/decryption keys, and SAML/OAuth token settings. Attackers can use the
+information extracted to forge malicious __VIEWSTATE requests for persistent RCE on the web server or pivot to the SQL
+server using exposed connection strings.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.file-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Unusual Web Config File Access"
```

Comment:
Is `web.config` a generalized filename or specific to Sharepoint & IIS? If not, maybe we add them in the rule name?

Reply:
It is specific to IIS/ASP.NET